src/libs/kata-types/src/capabilities.rs

```diff
@@ -17,6 +17,8 @@ pub enum CapabilityBits {
     MultiQueueSupport,
     /// hypervisor supports filesystem share
     FsSharingSupport,
+    /// hypervisor supports memory hotplug probe interface
+    GuestMemoryHotplugProbe,
```

Comment:
GuestMemoryProbe may be better?

Reply:
This is to know if the hypervisor supports Hotplug, maybe keep the keyword Hotplug is better here

Reply:
The path you read in agent is `/sys/devices/system/memory/probe`, leave only probe here is more precise. And you can also choose leaving only hotplug here, which is also make sense.

Reply:
fixed